fw/vhost.c

```diff
@@ -2056,6 +2069,52 @@ tfw_cfgop_frang_glob_set_int(TfwCfgSpec *cs, TfwCfgEntry *ce)
 	return tfw_cfg_set_int(cs, ce);
 }
 
+static int
+tfw_cfgop_frang_glob_set_long(TfwCfgSpec *cs, TfwCfgEntry *ce)
+{
+	if (ce->dflt_value && *(unsigned long *)(cs->dest))
+		return 0;
+	return tfw_cfg_set_long(cs, ce);
+}
+
+static int
+tfw_cfgop_frang_glob_http_methods(TfwCfgSpec *cs, TfwCfgEntry *ce)
+{
+	long int *dest_long = cs->dest;
+
+	if (ce->dflt_value && *(unsigned long *)(cs->dest))
+		return 0;
+	return __tfw_cfgop_frang_http_methods(cs, ce, dest_long);
+}
+
+static int
+tfw_cfgop_frang_glob_http_ct_vals(TfwCfgSpec *cs, TfwCfgEntry *ce)
+{
+	FrangVhostCfg *cfg = &tfw_frang_vhost_reconfig;
+
+	if (cfg->http_ct_vals) {
+		if (ce->dflt_value)
+			return 0;
+		kfree(cfg->http_ct_vals);
+		cfg->http_ct_vals = NULL;
+	}
+	return __tfw_cfgop_frang_http_ct_vals(cs, ce, cfg);
+}
+
+static int
+tfw_cfgop_frang_glob_rsp_code_block(TfwCfgSpec *cs, TfwCfgEntry *ce)
+{
+	FrangVhostCfg *cfg = &tfw_frang_vhost_reconfig;
```

Comment:
Why we can't use `tfw_cfgop_frang_rsp_code_block`. The only difference that here we directly set FrangVhostCfg *cfg = &tfw_frang_vhost_reconfig; instead of FrangVhostCfg *cfg = tfw_cfgop_frang_get_cfg(); ?

Reply:
It was attempt to solve strange location inheritance for default vhost. I returned it back now because my code had changed overriding.